lib/main.coffee

```diff
@@ -28,11 +33,37 @@ module.exports =
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
+        fileDir = _path2.default.dirname(filePath)
+
+        configFile = helpers.findCached fileDir, [
```

Comment:
Can you add searching `process.env` for `HTML_TIDY` for the absolute path? (don't bother with relative path handling for that).